app/grants/templates/grants/detail/info.html

```diff
@@ -81,7 +81,7 @@ <h1 class="font-title-xl my-4 font-weight-bold">
 
       <div class="mb-2">
         <i style="width: 14px;" class="fab fa-ethereum mr-2 text-center"></i>
-        <a id="wallet-address" class="wallet-address" href="https://etherscan.io/address/{{ grant.admin_address }}"
+        <a id="wallet-address" class="wallet-address"
```

Comment:
Should we use an anchor tag for this? https://www.w3.org/TR/html5/text-level-semantics.html#the-a-element

Reply:
It is an anchor tag ! The href gets populated [here](https://github.com/gitcoinco/web/pull/4720/files#diff-ce752a635ca0517ae9c9403aadbd5f09R94)

This ensures that if the grant gets created on mainnet -> the main-net etherscan link shows up 
. If it's rinkeby -> the rinkeby one shows up 

I can leave an empty href tag if that helps @danlipert